src/cartservice/src/cartservice.csproj

```diff
@@ -5,9 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.AspNetCore" Version="2.45.0" />
-    <PackageReference Include="Grpc.HealthCheck" Version="2.45.0" />
-    <PackageReference Include="StackExchange.Redis" Version="2.5.61" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.46.0" />
+    <PackageReference Include="Grpc.HealthCheck" Version="2.46.3" />
+    <PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="6.0.5" />
```

Comment:
This package is the official one maintained by Microsoft: https://www.nuget.org/packages/Microsoft.Extensions.Caching.StackExchangeRedis. It's encapsulating the `StackExchange.Redis` and it has the benefit of implementing a common interface `IDistributedCache` for different implementation like explained here: https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed?view=aspnetcore-6.0.